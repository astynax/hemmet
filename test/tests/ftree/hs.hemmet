|hs|src/{!lib/util data/text/extra} test/{lib-spec data/text/extra-spec}
