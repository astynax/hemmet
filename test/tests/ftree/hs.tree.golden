.
├── src/
│   ├── Data/
│   │   └── Text/
│   │       └── Extra.hs
│   ├── Lib/
│   │   └── Util.hs
│   └── Lib.hs
└── test/
    ├── Data/
    │   └── Text/
    │       └── ExtraSpec.hs
    └── LibSpec.hs
